docker-images/kafka/scripts/cruise_control_tls_prepare_certificates.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
there are two of this file?

Reply:
Yeah this appear to be a copy of the `cruise-control-scripts/cruise_control_tls_prepare_certificates.sh`